cli/src/main/java/com/liferay/blade/cli/WorkspaceConstants.java

```diff
@@ -50,6 +51,8 @@ public class WorkspaceConstants {
 
 	public static final String DEFAULT_WORKSPACE_PRODUCT_PROPERTY = "liferay.workspace.product";
 
+	public static final Pattern dxpQuarterReleaseVersionPattern = Pattern.compile(
```

Comment:
This is just asking to be broken in the future.  Can we do without this pattern?

Reply:
yes, we could. I will change it.